packages/curve-ui-kit/src/features/market-details/index.tsx

```diff
@@ -139,15 +204,29 @@ export const MarketDetails = ({
           tokenAddress={borrowToken?.tokenAddress}
           loading={borrowToken?.symbol == null && borrowToken?.loading}
           blockchainId={blockchainId}
+          valueTooltip={{
+            title: t`Debt Token ${marketType === 'lend' ? t`(Lending Markets)` : t`(Mint Markets)`}`,
+            body: <DebtTokenTooltip marketType={marketType} />,
+            placement: 'top',
+            arrow: false,
+            clickable: true,
+          }}
         />
         {/* Insert empty box to maintain grid layout when there is no lending APY metric */}
-        {!lendingAPY && <Box />}
+        {!supplyAPY && <Box />}
         <Metric
           size="small"
           label={t`Available liquidity`}
           value={availableLiquidity?.value}
           loading={availableLiquidity?.value == null && availableLiquidity?.loading}
           valueOptions={{ unit: 'dollar' }}
+          valueTooltip={{
+            title: t`Available Liquidity ${marketType === 'lend' ? t`(Lending Markets)` : t`(Mint Markets)`}`,
```

Comment:
duplicated code :upside_down_face: 
```suggestion
            title: t`Available Liquidity ${marketTypeSuffix}`,
```